LayoutTests/fast/dom/normalize-doesnt-check-string-length.html

```diff
@@ -0,0 +1,25 @@
+<html>
+<head>
+    <script src="../../resources/js-test.js"></script>
+    <script>
+        function f1() {
+            const MAX_LENGTH = 1 << 28;
+
+            const largeString = 'A'.repeat(MAX_LENGTH);
+
+            const container = document.getElementById('container');
+
+            for (let i = 0; i < 9; ++i)
+                container.appendChild(document.createTextNode(largeString));
+
+            shouldThrowErrorName('container.normalize()', 'InvalidModificationError');
+        }
+
+        window.testRunner?.dumpAsText();
+    </script>
+</head>
+<body onload="f1()">
+    <div id="container" hidden></div>
+    <p>This test passes if webkit doesn't crash and there's an exception</p>
```

Comment:
Please call `description("This test passes if webkit doesn't crash and there's an exception");` in JavaScript instead.